Offseason2022/src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -475,18 +479,14 @@ private double nativeUnitsToMPS(double nativeUnitsVelocity)
 
   private double getDistanceMetersLeft( )
   {
-    if (m_validL1)
-      return nativeUnitsToMeters(m_driveL1.getSelectedSensorPosition(PIDINDEX));
 
-    return 0;
+    return (m_validL1) ? (m_offset1 + nativeUnitsToMeters(m_driveL1.getSelectedSensorPosition(PIDINDEX))) : 0;

```

Comment:
Nice and concise. Might want to review what this line is doing with the rest of the group.